### PR TITLE
Completed tests for number Utility.

### PR DESCRIPTION
**Pull request by ocampd1**

### CODE REVIEW

File: chucks-pick3-mobile/src/app/pages/home/home.page.spec.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { IonicModule } from '@ionic/angular';
+
+import { HomePage} from "./home.page";
+
+describe('HomePage', () => {
```
Comment (pouncilt) [inline chucks-pick3-mobile/src/app/pages/home/home.page.spec.ts:6]: Give unique Test suite name.

File: chucks-pick3-mobile/src/app-vanilla/home-vanilla/home-vanilla.page.spec.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
 import { IonicModule } from '@ionic/angular';
 
 import { HomeVanillaPage } from './home-vanilla.page';
+import {CardContextService} from '../../app/services/card-context.service';
 
 describe('HomePage', () => {
```
Comment (pouncilt) [inline chucks-pick3-mobile/src/app-vanilla/home-vanilla/home-vanilla.page.spec.ts:7]: Give unique Test suite name.

**Review by pouncilt — CHANGES_REQUESTED** [review]

File: chucks-pick3-mobile/src/app/pages/home/home.page.spec.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      declarations: [ HomePage ],
+      imports: [IonicModule.forRoot()]
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(HomePage);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  }));
+
+  it('should create', () => {
```
Comment (pouncilt) [inline chucks-pick3-mobile/src/app/pages/home/home.page.spec.ts:21]: This test is failing.  Either fix or skip for now.

**Review by pouncilt — CHANGES_REQUESTED** [review]
Skip or fix failing tests

File: chucks-pick3-mobile/src/app-vanilla/home-vanilla/home-vanilla.page.spec.ts (hunk 2 above)
Comment (ocampd1) [reply chucks-pick3-mobile/src/app-vanilla/home-vanilla/home-vanilla.page.spec.ts]: Resolved naming convention.

File: chucks-pick3-mobile/src/app/pages/home/home.page.spec.ts (hunk 3 above)
Comment (ocampd1) [reply chucks-pick3-mobile/src/app/pages/home/home.page.spec.ts]: Resolved test to skip.

File: chucks-pick3-mobile/src/app/pages/home/home.page.spec.ts (hunk 1 above)
Comment (ocampd1) [reply chucks-pick3-mobile/src/app/pages/home/home.page.spec.ts]: Resolved naming convention.

**Pull request merged**